tools/bug-report/pkg/cluster/cluster.go

```diff
@@ -0,0 +1,166 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cluster
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	v1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"istio.io/pkg/log"
+)
+
+type ResourceType int
+
+const (
+	Namespace ResourceType = iota
+	Deployment
+	Pod
+	Label
+	Annotation
+	Container
+)
+
+func ParsePath(path string) (namespace string, deployment, pod string, container string, err error) {
+	pv := strings.Split(path, "/")
+	if len(pv) != 4 {
+		return "", "", "", "", fmt.Errorf("bad path %s, must be namespace/pod/container", path)
```

Comment:
is deployment missing from the err message? also add expected format to comments

Reply:
done.